textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexToGramQueryTranslatorTest.java

```diff
@@ -10,6 +12,16 @@
 
 public class RegexToGramQueryTranslatorTest {
 	
+	private void printTranslatorResult(String regex) {
+		GramBooleanQuery exactQuery = RegexToGramQueryTranslator.translate(regex);
+		
+		System.out.println("regex: "+regex);
```

Comment:
We are using the function to test the equivalence of two boolean expressions. As you could see in first half of the test cases, we get `exactQuery` by calling `translate`, hardcode `expectedQuery`, and assert them equal.
But as the regex gets more complex, the complexity of expected query increases significantly. It's a bit messy to hardcode all the way to build expectedQuery. More importantly, since the expected query is no longer straightforward, we are literally copying from exact query to build expected query. So we think it does not make much sense to build expected query and assert it.

Having said that, we should figure out a way to automate the test process. For now println() is for debugging and displaying the result of this implementation.